app/views/profile/favorites/index.html.erb

```diff
@@ -1,15 +1,21 @@
 <h2>My Favorites</h2>
-
-<% if @recipes.nil? %>
-  <p>You have not favorited any recipes.</p>
-<% else %>
-  <% @recipes.each do |recipe| %>
-    <div id="recipe-<%= recipe.id %>">
-      <%= link_to recipe_path(recipe.id) do %>
-        <%= recipe.title %><br>
+<div class="container">
+  <div class="row">
+    <% unless @recipes.nil? %>
+    <% @recipes.each do |recipe| %>
+    <div class="col-sm-4">
+      <div class="recipe-<%= recipe.id %>">
```

Comment:
May want to consider switching this class for an ID since each div generated by the enumerable is unique.